transforms/create-element-to-jsx.js

```diff
@@ -66,11 +66,23 @@ module.exports = function(file, api, options) {
     }
   };
 
+  const jsxIdentifierFor = node => {
```

Comment:
@cpojer as you can see, the method now returns a node, but the recursion is still there, as we need to be able to convert `React.createElement(A.b.c.d)` as well. So `node.object` can be another `MemberExpression` that needs to be converted to a `JsxMemberExpression`, and `node.object` is an Identifier that should be converted to a `jsxIdentifier`. I could probably just change that for `j.jsxIdentifier(node.name)` but that would technically be duplicating the `else` block.

I hope I understood your request ;-) let me know what you think!